src/cascadia/TerminalCore/TerminalApi.cpp

```diff
@@ -615,6 +615,10 @@ void Terminal::UseAlternateScreenBuffer()
     // update all the hyperlinks on the screen
     UpdatePatternsUnderLock();
 
+    // GH#3321: Make sure we let the TerminalInput know that we switched
+    // buffers. This might affect how we interpret certain mouse events.
+    _terminalInput->UseAlternateScreenBuffer();

```

Comment:
IMHO this part belongs in the original alt buffer PR -- since `UseAlternateScreenBuffer` isn't new.